src/main/java/codeu/model/data/Conversation.java

```diff
@@ -43,18 +42,6 @@ public Conversation(UUID id, UUID owner, String title, Instant creation) {
     this.owner = owner;
     this.creation = creation;
     this.title = title;
-    tags = new ArrayList<>();
-  }
-
-  /** Adds a tag to this Conversation. */
-  public void addTag(Tag tag) {
-    tags.add(tag);
-  }
-
-  /** Returns an alphabetically sorted list of Tags */
-  public List<Tag> getTags() {
-    tags.sort(new TagComparator());
-    return tags;
```

Comment:
Why is this all deleted? A Conversation can have multiple Tags.

Reply:
I took the references to Tags out of Conversations for now,,, There's probably a better way but for now adding tags to a Conversation and storing them in the list was messing up the ConversationStore (i.e. adding the same Conversation in memory multiple times)